arches_lingo/src/arches_lingo/components/scheme/report/SchemeNamespace.vue

```diff
@@ -52,6 +58,7 @@ async function save() {
                       ),
         });
     }
+    emits("updated");
```

Comment:
nit: emits uses the declaration while this uses the string. Also IMO this should go in the `try`